src/components/shared/DivWithTooltip.js

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+import * as React from 'react';
+import Tooltip from './Tooltip';
+import type { CssPixels } from '../../types/units';
+
+type Props = {
+  tooltip: React.Node,
+  children?: React.Node,
+};
```

Comment:
nit: make it stricter with | and + ?

Reply:
I can't add `|` because the object contains the unknown `containerProps` as well. I added the `+`.